File: java-boot/src/main/java/se/callistaenterprise/kotlindsl/DslApplication.java
package se.callistaenterprise.kotlindsl;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.r2dbc.R2dbcAutoConfiguration;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication(exclude = { R2dbcAutoConfiguration.class })
@EnableScheduling
public class DslApplication {

    public static void main(String[] args) {
        SpringApplication.run(DslApplication.class, args);
    }

}

File: kotlin-boot/src/main/kotlin/se/callistaenterprise/kotlindsl/DslApplication.kt
package se.callistaenterprise.kotlindsl

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.r2dbc.R2dbcAutoConfiguration
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication(exclude = [R2dbcAutoConfiguration::class])
@EnableScheduling
class DslApplication

fun main(args: Array<String>) {
    runApplication<DslApplication>(*args)
}
